apps/web/modules/getting-started/[[...step]]/onboarding-view.tsx

```diff
@@ -75,18 +64,6 @@ const OnboardingPage = (props: PageProps) => {
     {
       title: `${t("connect_your_video_app")}`,
       subtitle: [`${t("connect_your_video_app_instructions")}`],
-      skipText: `${t("set_up_later")}`,
```

Comment:
no longer needed because "Connect your video" page will be the last step for those who get landed onto the page